src/main/java/BankingSystem.java

```diff
@@ -0,0 +1,209 @@
+import dao.AccountDao;
+import model.Account;
+import model.Card;
+import model.CardGenerator;
+
+import java.util.Scanner;
+
+public class BankingSystem {
```

Comment:
Can you use the default access modifier?

Reply:
Changed